### DISCUSSION TITLE
I'm migrating V9 -> V10. Can I replace `useContext()` with `useUtils()` ?

### DISCUSSION THREAD

**Discussion by mannerism** [trpc/trpc#5132] [Q&A] [answered]
Hello all, 

Environment: 
Typescript, TRPC, Tailwind, Node, Prisma, NextJS

I've been using `useContext()` to invalidate a certain route in the client so that I can refresh the page conveniently. Here is the actual code I'm using in the client:

```
  const ctx = api.useContext();

  const { mutate: approve, isLoading: isApproving } =
    api.enrollment.approve.useMutation({
      onSuccess: () => {
        void ctx.sprints.getSprintByIdAdmin.invalidate();
        toast.success("successfully approved enrollment");
        setSelectedReq(undefined);
      },
      onError: (e) => {
        toast.error(e.message);
      },
    });
```

Can I replace `useContext()` which is now deprecated in the version 10 to `useUtils()` ?

Thank you.

**Comment by KATT** [comment, marked as answer]
Yes
